test/test_helm_python_django_app.py

```diff
@@ -22,62 +44,22 @@ def setup_method(self):
     def teardown_method(self):
         self.hc_api.delete_project()
 
-    @pytest.mark.parametrize(
-        "version,branch",
-        [
-            ("3.12-ubi9", "4.2.x"),
-            ("3.12-ubi8", "4.2.x"),
-            ("3.11-ubi9", "4.2.x"),
-            ("3.11-ubi8", "4.2.x"),
-            ("3.9-ubi9", "master"),
-            ("3.9-ubi8", "master"),
-            ("3.6-ubi8", "master"),
-        ],
-    )
-    def test_django_application_curl_output(self, version, branch):
-        if self.hc_api.oc_api.shared_cluster:
-            pytest.skip("Do NOT test on shared cluster")
-        self.hc_api.package_name = "python-imagestreams"
-        assert self.hc_api.helm_package()
-        assert self.hc_api.helm_installation()
-        self.hc_api.package_name = "python-django-application"
-        self.hc_api.helm_package()
-        assert self.hc_api.helm_installation(
-            values={
-                "python_version": version,
-                "namespace": self.hc_api.namespace,
-                "source_repository_ref": branch,
-            }
-        )
-        assert self.hc_api.is_s2i_pod_running(pod_name_prefix="django-example")
-        assert self.hc_api.test_helm_curl_output(
-            route_name="django-example",
-            expected_str="Welcome to your Django application"
-        )
-
-    @pytest.mark.parametrize(
-        "version,branch",
-        [
-            ("3.12-ubi9", "4.2.x"),
-            ("3.12-ubi8", "4.2.x"),
-            ("3.11-ubi9", "4.2.x"),
-            ("3.11-ubi8", "4.2.x"),
-            ("3.9-ubi9", "master"),
-            ("3.9-ubi8", "master"),
-            ("3.6-ubi8", "master"),
-        ],
-    )
-    def test_django_application_helm_test(self, version, branch):
-        self.hc_api.package_name = "python-imagestreams"
+    def test_django_application_helm_test(self):
+        new_version = VERSION
+        if "minimal" in VERSION:
+            new_version = VERSION.replace("-minimal", "")
+        if OS == "rhel10":
+            pytest.skip("Do NOT test on rhel10. It is not released yet.")
```

Comment:
I'd do this as the first thing in the function.

Reply:
Fixed by https://github.com/sclorg/s2i-python-container/pull/734/commits/2d4a4f8d34010f7ca89b9a8e236906d0f59c9369